products/apache-activemq.md

```diff
@@ -24,6 +24,12 @@ auto:
 # - y <= 9 and z > 9 : "https://activemq.apache.org/activemq-{{'__LATEST__'|replace_first:'.','00'|replace_first:'.','0'}}-release"
 # - y <= 9 and z <= 9 : "https://activemq.apache.org/activemq-{{'__LATEST__'|replace:'.','00'}}-release"
 releases:
+-   releaseCycle: "6.0"
+    releaseDate: 2023-11-12
+    eol: false # still listed on https://activemq.apache.org/
+    latest: "6.0.0"
+    latestReleaseDate: 2023-11-12
```

Comment:
```suggestion
    releaseDate: 2023-11-17
    eol: false # still listed on https://activemq.apache.org/
    latest: "6.0.0"
    latestReleaseDate: 2023-11-17
```

According to https://activemq.apache.org/activemq-6000000-release. Will be overwritten by automation, but at least would be in the git history.